src/components/hint/index.tsx

```diff
@@ -613,7 +614,7 @@ const styles = StyleSheet.create({
     position: 'absolute'
   },
   hint: {
-    maxWidth: 400,
+    maxWidth: DEFAULT_MAX_WIDTH,
```

Comment:
Technically it's not default, the user cannot override it; I'd even consider not adding a const:
```
maxWidth: Math.min(Constants.screenWidth - 2 * Spacings.s4, 400),
```

Reply:
done